sdk/core/Azure.Core/tests/public/PatchModels/ParentPatchModelTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Serialization;
+using Azure.Core.Tests.PatchModels;
+using NUnit.Framework;
+
+namespace Azure.Core.Tests.Public.ModelSerializationTests
+{
+    internal class ParentPatchModelTests : ModelJsonTests<ParentPatchModel>
+    {
+        protected override string JsonPayload => """
+        {
+            "id": "abc",
+            "child": {"a": "aa", "b": "bb"}
```

Comment:
I would add an extra property here that isn't defined in the model to validate that portion of the serialization expectations as well.

Reply:
That feels out of scope for this PR - is there a reason it is needed here?

Reply:
Since we are defining a new model template that the generator would need to create I think its important to make sure it not only enables the new functionality but is compatible with all the existing functionality.

Its pretty simple to do just add `"extra": "stuff"` to the payload and in your expected overload you just check if format is `J` then that should be there.

Example change to expected https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/tests/public/ModelSerializationTests/ModelXTests.cs#L42-L43.

Example change to verification https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/tests/public/ModelSerializationTests/ModelXTests.cs#L54-L58